doc/specs/#4191 - Formatted Copy/spec.md

```diff
@@ -1,7 +1,7 @@
 ---
 author: Kayla Cinnamon @cinnamon-msft
 created on: <2020-04-01>

```

Comment:
the reason these are in `<>` in the spec template is to indicate that the entire value is a placeholder and should be changed. The `<>` should not be retained in the actual spec document.